README.md

```diff
@@ -39,12 +39,14 @@ An Airflow container can be built with
 make build
 ```
 
-You should then run the database migrations to complete the container initialization with
+You should then initialize containers with:
 
 ```bash
-make migrate
+nohup make up &
```

Comment:
Maybe mention that `make clean` is the appropriate way stop the backgrounded service

Reply:
rm'd.  The make has a help prompt and the readme mentions make clean later